transform/app/transform/utils/common.py

```diff
@@ -664,47 +664,46 @@ def harvest_exportation(df: DataFrame, harvested_properties: dict) -> None:
     The extracted information is structured into a list of dictionaries for each instance and stored in
     'harvested_properties[EXPORTATION]'.
     Note:
-    - 'i' is used to limit harvesting to the first 10 versions of each instance.
+    - 'instance_idx' is used to limit harvesting to the first 10 versions of each instance.
     """
     instances_list = df.select(INSTANCE).collect()
     exportation_column = []
+    instances_limit = 10
 
     for instances in instances_list:
         if instances[INSTANCE]:
             exportation_row = []
 
-            for i, instance in enumerate(instances[INSTANCE]):
-                if i >= 9:
+            for instance_idx, instance in enumerate(instances[INSTANCE], start=1):
+                if instance_idx > instances_limit:
```

Comment:
I would rather write:
`if instance_idx >= instances_limit:` without `start=1`. Sorry my bad, I wrote bad condition in a previous comment

Reply:
No problemo, already corrected.